### PR TITLE
Ce style item list

### LINKED ISSUE #13: 13. As a user, I want the app to look professional and welcoming
The last 2 weeks of the project will function a little differently than the rest. Your team will **self-organize** around a lightweight **design process** to come up with a polished look-and-feel for your app.
      This will require you to **make decisions together as a whole team,** then implement your vision. The process outlined here is our best suggestion for how to proceed, but **you are free to take whatever approach you see fit!**
      You’ll notice this is the last issue on the board! That’s because the expectation is for you to **write the stories needed to complete the implementation** of your design direction.
      As always, **lean on your mentors** for guidance if you are unsure how to proceed!
      ## Design Process
      1. What is the tone you’re going for? What feeling are you trying to convey? Come up with 3 to 5 words that best capture how you want users to perceive your app.
          - Examples:
              - Fun
              - Clean
              - Ironic
              - Businesslike
              - [Many more example words](http://static1.squarespace.com/static/5723ff8086db4345e722e7be/573248885f21e0a9e45338ef/57b6a206579fb39bc0e08e2a/1471586826621//img.jpg)
      2. Create a [mood](https://milanote.com/guide/moodboard) [board](https://www.canva.com/learn/make-a-mood-board/). A mood board helps everyone get on the same page in terms of the direction in which you want to take your design. Add elements to your mood board that represent the tone you’re trying to achieve.
      3. Decide on fundamental elements of your design, including the following:
          - **Typography** ← [Google Fonts](https://fonts.google.com/) is a good choice for finding web fonts you can use in your app. They have hundreds of typefaces to choose from, but consider usability in whichever one(s) you choose!
          - **Colors** ← Make sure these are [accessible](https://accessible-colors.com/)!
          - **Iconography / imagery** ← Start here to [find free icons you can use in your app](https://www.canva.com/learn/free-icons-download/)!
      4. Decide whether you want to code your CSS from scratch or integrate a CSS framework. There are pros and cons to each!
          - Vanilla CSS gives you more control, but it can be harder and/or take more time to achieve a given effect.
          - Frameworks help you get things done faster, but often at the cost of not giving you full control of the design.
      5. If you choose to use a framework, evaluate your options to make sure you pick one that will suit your needs.
          - Consider the following:
              - Quality of documentation
              - Ease of integration
              - Built with accessibility in mind
              - Activeness of the community
          - Quality frameworks you might consider include the following:
              - [Bootstrap](https://getbootstrap.com/)
              - [Chakra UI](https://chakra-ui.com/)
              - [Material UI](https://material-ui.com/)
              - [Semantic UI](https://semantic-ui.com/)
              - [TailwindCSS](https://tailwindcss.com/)

### LINKED ISSUE #40: 13c: As a user, I want to be able to press buttons that clearly show which action they perform.
AC: 

- Style the following buttons using the text/icons per the design in Figma:
		
			- Delete Item button [round red X icon]
			- Item Purchased [round + icon, color coordinates with each frequency category color]
			- Add Item button [add words above question mark img or separate from img? tbd]
			- Add color coordination to the frequency options in Add Item form

- Add hover and active state styling to all buttons

### LINKED ISSUE #43: 13b: As a developer, I want to create some reusable design components for my app.
This issue will set up the foundations of the overall design: background, layout, typography, colors, and text buttons.
AC:
- Style the Homepage according to the design in Figma, including but not limited to: 
- setting the background image to [chalkboard image outlined in Figma]
- adding the logo to the top of the layout
- setting the fonts for each section [font A for headings, font B for items in the list and messages]
- set the font color to [white]
- set the container height/width parameters for mobile/desktop 
- the join list / token form should have logo/flourishes per the design
- styling the primary text buttons that will be used throughout the app with padding, margins, hover, and active states

### PR DESCRIPTION
**Pull request by cynthiaeddy**
## Description

This PR creates reusable design components for the app.

## AC
Style the Homepage according to the design in Figma, including but not limited to:
  	- setting the background image to [chalkboard image outlined in Figma]
  	- adding the logo to the top of the layout
  	- setting the fonts for each section [font A for headings, font B for items in the list and messages]
  	- set the font color to [white]
  	- the join list / token form should have logo/flourishes per the design
  	- styling the primary text buttons that will be used throughout the app with padding, margins, hover, and active states
Caveat: Following aren't completed i.e. breakpoints
  	- set the container height/width parameters for mobile/desktop 

## Related Issue 
Closes #43 

## Type of Changes
|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|    | :sparkles: New feature     |
|  ✓ | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

### Before and After

| opener before            |  opener after |  add item before        |  add item after | 
:-------------------------:|:-------------------------:|:-------------------------:|:-------------------------:
Screen Shot 2022-03-01 at 6 18 23 PM | Screen Shot 2022-03-01 at 7 25 38 PM |Screen Shot 2022-03-01 at 6 18 46 PM| Screen Shot 2022-03-01 at 6 17 22 PM
| list item before            |  list item after |         |          | 
Screen Shot 2022-03-01 at 6 18 55 PM | Screen Shot 2022-03-01 at 6 17 32 PM     

## Testing Steps / QA Criteria

Pull down the branch and compare styling to the [Figma](https://www.figma.com/file/dsTg5iRxSMbchiMjYT8mad/collab?node-id=0%3A1) mobile design for the home page. Please note that the desktop styling has not been added yet.

### CODE REVIEW

**Comment by diyanamendoza** [comment]
Looking so slick. Some questions below! Happy to discuss further at office hours:
1. Should we incorporate a burger menu in the top right or left so that we can include the about link there, and conditionally show the add item and item list nav links if there is a token? I am happy to build this.
2.  On the add Item page, how do we feel about adding some user feedback (like underline or whatnot) upon selection of 'soon,' 'kind of soon,' 'not soon' so it's clear which one was chosen? 
3. Also on the add item page, it wasn't immediately clear to me that the question mark was the way to submit my item. Can we change that to a button that clearly says add?

**Comment by cynthiaeddy** [comment]
thanks D! - please look at the figma docs, i hope they will answer your questions about the add item page, as far as user navigation.
I was thinking that upon entering the item name, the soon, kind of soon, not so soon are in their coordinated colors. when you hover over each name, it turns white, upon activating (clicking on the said name) the active name stays it's color, while the other ones fade to 50%.  i know this pr doesn't fully achieve said parameters - just wanted to get it merged with main. I also would like once the user clicks on their option, for the page to set a timeout? in order for it to render 'hurray! you've added blah blah to your list like the figma docs

also - great call about hamburger menu - i love em!

**Review by diyanamendoza — APPROVED** [review]

**Review by k-monnik — APPROVED** [review]
Great work!

**Review by chriskorsak — APPROVED** [review]
Loving the design. Nice job team! Looking forward to the final product!

File: postcss.config.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
-    tailwindcss: {},
-    autoprefixer: {},
-  },
-};
+// module.exports = {
+//   plugins: {
+//     tailwindcss: {},
+//     autoprefixer: {},
+//   },
+// };
```
Comment (aprilleone) [inline postcss.config.js:6]: Is there a plan to put this back in again? If not, it should be removed.

File: src/App.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import AddItem from './components/AddItem';
 import ItemList from './components/ItemList';
 import Home from './pages/Home';
-import Nav from './components/Nav';
+// import Nav from './components/Nav';
 import { TokenProvider } from './context/TokenContext';
```
Comment (aprilleone) [inline src/App.js:8]: is there a plan to have this back in? If not, it should be deleted.

File: src/components/AddItem.js [hunk 3]
```diff
@@ -63,41 +68,74 @@ const AddItem = () => {
...
+        alt="Logo: Welcome to Your Smart Shopping List"
+        className="logo"
+      />
+      <div className="outer-box">
+        <div className="inner-box">
+          <form onSubmit={handleSubmit} className="">
+            <label
+              htmlFor="item-name"
+              className="bg-gray-800 pr-2 pl-2 absolute left-1/2  -mt-4 text-xs uppercase tracking-wider transform -translate-x-1/2 text-white/80"
+            >
```
Comment (aprilleone) [inline src/components/AddItem.js:82]: woohoo tailwind!

**Review by aprilleone — APPROVED** [review]
minor nits, not blocking.

**Pull request merged**